burnman/tools.py

```diff
@@ -643,3 +644,109 @@ def check_eos_consistency(m, P=1.e9, T=300., tol=0.01, verbose=False):
             print('Not satisfied all EoS consistency constraints for {0:s}'.format(m.to_string()))
             
     return consistency
+
+
+def _pad_ndarray_inverse_mirror(array, padding):
+    """
+    Pads an ndarray according to an inverse mirror
```

Comment:
reflect and mirror are essentially the same; the boundaries are treated as follows:
           pad | array
reflect: 4 3 | 3 4 6
mirror:  6 4 | 3 4 6
newfn:  0 2 | 3 4 6